source/framework/vx_node.c

```diff
@@ -1841,14 +1847,20 @@ VX_API_ENTRY vx_status VX_API_CALL vxReplicateNode(vx_graph graph, vx_node first
                         }
                     }
                 }
-                vxReleaseReference(&ref);
-                vxReleaseParameter(&param);
             }
             else
             {
                 VX_PRINT(VX_ZONE_ERROR, "Parameter %d is NULL!\n", p);
                 status = (vx_status)VX_FAILURE;
             }
+            if (ref)
```

Comment:
MISRAC will want this to be if ((vx_reference)NULL != ref)

Reply:
done, but are you sure this is something we modified?

Reply:
done in the latest commits